src/main/scala/cromwell/util/google/GoogleScopes.scala

```diff
@@ -1,12 +1,17 @@
 package cromwell.util.google
 
+import com.google.api.services.genomics.GenomicsScopes
 import com.google.api.services.storage.StorageScopes
+import scala.collection.JavaConverters._
 
 object GoogleScopes {
-  val Scopes = Vector(
-    StorageScopes.DEVSTORAGE_FULL_CONTROL,
-    StorageScopes.DEVSTORAGE_READ_WRITE,
-    "https://www.googleapis.com/auth/genomics",
-    "https://www.googleapis.com/auth/compute"
-  )
+  val Scopes = {
+    val scopes = Vector(
+      StorageScopes.DEVSTORAGE_FULL_CONTROL,
+      StorageScopes.DEVSTORAGE_READ_WRITE,
+      "https://www.googleapis.com/auth/genomics",
+      "https://www.googleapis.com/auth/compute"
+    ) ++ GenomicsScopes.all().asScala
```

Comment:
Check the previous change by @tovanadler [here](https://github.com/broadinstitute/cromwell/commit/3601d2f6c8517db3e27fd5c88ecd80479032e7bf#diff-237dfbe12dc36741c382f3651aa7f1beR162), in the first commit. I still haven't mastered scopes, so kept it.